### PR TITLE
HS-603: Add Event gRPC Service to query events

### PR DESCRIPTION
**Pull request by thomasbigger584**
## Description
HS-603: Add Event gRPC Service to query events

## Jira link(s)
- https://issues.opennms.org/browse/HS-603

## Checklist
* [x] Follows Horizon Stream's [development guidelines.](https://github.com/OpenNMS/horizon-stream/wiki/Development-Guidelines)
* [x] Appropriate reviewer(s) have been selected.
* [x] Jira issue(s) have been updated to "In Review".
* [x] Includes [appropriate tests.](https://github.com/OpenNMS/horizon-stream/wiki/Test-Strategy)
* [ ] Documentation has been updated as necessary.

### CODE REVIEW

File: shared-lib/events/src/main/proto/events.proto [hunk 1]
```diff
@@ -67,3 +70,23 @@ message EventParameter {
   string type = 3;
   string encoding = 4;
 }
+
+message EventDTO {
```
Comment (cgorantla) [inline shared-lib/events/src/main/proto/events.proto:74]: I think we should separate grpc events api from kafka events proto.

Comment (thomasbigger584) [reply shared-lib/events/src/main/proto/events.proto:74]: seperate this into another proto file?

Comment (cgorantla) [reply shared-lib/events/src/main/proto/events.proto:74]: yes, we need to move EventDTO into separate proto file.  don't need to depend on the same EventParameter/EventInfo

Comment (thomasbigger584) [reply shared-lib/events/src/main/proto/events.proto]: done

File: events/persistence/src/main/java/org/opennms/horizon/events/persistence/mapper/EventMapper.java [hunk 2]
```diff
@@ -0,0 +1,56 @@
+package org.opennms.horizon.events.persistence.mapper;
```
Comment (cgorantla) [inline events/persistence/src/main/java/org/opennms/horizon/events/persistence/mapper/EventMapper.java:1]: missing copyright headers

**Review by cgorantla — APPROVED** [review]
lgtm!

**Pull request merged**